cips/cip-3.md

```diff
@@ -0,0 +1,66 @@
+---
+cip: 3
+title: Diamonds Paid In CLOUT
+author: RedPartyHat <@redpartyhat>
+discussions-url: https://github.com/bitclout/cips/discussions/26
+created: 2021-08-04
+---
+
+## One-Line Description
+
+Pay for diamonds with CLOUT instead of creator coins.
+
+## Explanation and Motivation
+
+"Diamonds" are a tool that allow users to show their love for a creator's work on BitClout. A user
+can give up to six diamonds per post and each diamond given comes with a reward for the creator. A
+single diamond gives the creator ~0.005 USD (half a penny) at the time of writing. Each additional
+diamond 10x's the reward, up to ~$500 USD for six diamonds. This has already become a significant
+source of income for many creators.
+
+Currently, the reward given with each diamond is paid in the giver's creator coin. This is great
+for creating a dense creator coin ownership graph. Diamond recipient's often do not sell the creator
```

Comment:
What can be done to implement other means to achieve a dense creator coin ownership graph when diamonds switch to being paid in CLOUT?

Reply:
@redpartyhat @diamondhands0 ive not seen a response on this point ? 

i think its one of the biggest benefits of cc diamonds vs clout diamonds.

i know you are under time pressure to ship clearly but would have been good to see you attempt to address this ?